modules/virt-accessing-vmi-yaml-ssh.adoc

```diff
@@ -0,0 +1,133 @@
+
+The following examples show the configurations for the VMI's YAML file and the service YAML file.
+
+.Prerequisites
+* Install the OpenShift CLI (`oc`).
+* You must be in the same project as the VMI.
+
+.Procedure
+. In the YAML file for the VMI, add the label and a value for exposing the service for SSH connections. Enable the `masquerade` feature for the interface.
+
++
+.Example output
+[source,yaml]
+----
+...
+kind: VirtualMachineInstance
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-ssh <1>
+  name: vmi-ssh
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+      interfaces:
+      - masquerade: {} <2>
+        name: testmasquerade <3>
+      rng: {}
+    machine:
+      type: ""
+    resources:
+      requests:
+        memory: 1024M
+  networks:
+  - name: testmasquerade
+    pod: {}
+...
+----
+<1> This label is used by the service to identify the virtual machine instances that are enabled for SSH traffic connections. The label can be any `key:value` pair that is added as a `label` to this YAML file and as a `selector` in the service YAML file.
+<2> The interface type is `masquerade`.
+<3> The name of this interface is `testmasquerade`.
+
+. Create the VMI:
++
+[source,terminal]
+----
+oc create -f <path_for_the_VMI_YAML_file>
+----
++
+. In the YAML file for the service, specify the service name, port number, and the target port.
+
++
+.Example output
+[source,yaml]
+----
+...
+apiVersion: v1
+kind: Service
+metadata:
+  name: svc-ssh <1>
+spec:
+  ports:
+      targetPort: 22 <2>
+      protocol: TCP
+  selector:
+    special: vmi-ssh <3>
+  type: NodePort
+...
+----
+<1> Name of the SSH service.
+<2> The target port number for the SSH connection.
+<3> The label name must match the label specified in the YAML file for the VMI.
+
+. Create the service:
++
+[source,terminal]
+----
+oc create -f <path_for_the_service_YAML_file>
+----
++
+. Verify that the VMI is running:
++
+[source,terminal]
+----
+oc get vmi
+----
++
+.Example output
+[source,terminal]
+----
+NAME    AGE     PHASE       IP              NODENAME
+vmi-ssh 6s      Running     10.244.196.152  node01
+----
+
+. Check the service to find out which port was acquired by the service:
++
+[source,terminal]
+----
+$ oc get svc
+----
++
+.Example output
+[source,terminal]
+----
+NAME            TYPE       CLUSTER-IP     EXTERNAL-IP   PORT(S)           AGE
+svc-ssh-exp     NodePort   10.106.236.208 <none>        20022:30093/TCP   22s
+----
++
+In this example, the service acquired the port number 30093.
+
+. Log in to the VMI via SSH. Use the IP address of the node and the port number that display when the `oc get svc` command is used. The following example shows the `ssh` command with the node's IP address and the port number.
```

Comment:
"Use the IP address of the node and the port number that display when the `oc get svc` command is used."
I am afraid this phrasing doesn't fit perfectly.
You might want to consider being consistent with the same step in the example of
[Accessing a virtual machine instance via SSH](https://deploy-preview-31224--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-accessing-vm-consoles.html#virt-accessing-vmi-ssh_virt-accessing-vm-consoles):
`Use ... the port that you found in the previous step:`

Reply:
This paragraph and the sentences were already finetuned and approved during the SME and QE review.

Reply:
I am also from CNV Network QE (Like Meni), so my comments derive from the QE review I did.
Whether to address these comments or not is your call.

Reply:
Resolved per my note.